Client/Frontend/Browser/HomePanel/FavoritesViewController.swift

```diff
@@ -375,10 +358,32 @@ class FavoritesViewController: UIViewController, Themeable {
     
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
         super.traitCollectionDidChange(previousTraitCollection)
-        
+
+        updateConstraints()
         collection.collectionViewLayout.invalidateLayout()
     }
     
+    private func updateConstraints() {
+        let isIpad = UIDevice.isIpad
+        let isLandscape = view.frame.width > view.frame.height
```

Comment:
This is not enough for split screen on iPads, see screenshot.

We can either extend the collection width to take 3/4 of the screen in such cases or make favorites cell to be more dynamic. Might require a discussion with the design team

<img width="959" alt="Zrzut ekranu 2020-01-8 o 11 58 59" src="https://user-images.githubusercontent.com/6950387/71977335-d0171900-3218-11ea-971d-74fc48a5c125.png">